src/ct_dev_agent_delegation_mcp/opencode_client.py

```
@@ -0,0 +1,610 @@
+            return
+        
+        # Try environment
+        if url := os.getenv("OPENCODE_SERVER_URL"):
+            self.base_url = url
+            return
+        
+        # Try config file
+        config_file = Path.home() / ".opencode" / "config.json"
+        if config_file.exists():
+            try:
+                config = json.loads(config_file.read_text())
```

Ruff (0.13.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)